QuizApp/TokenStorage/SecureStorage.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import Keychain
+
+class SecureStorage {
+
+    private let keychain = Keychain()
+
+    var accessToken: String? {
+        keychain.value(forKey: "accessToken") as? String ?? ""
```

Comment:
Make `accessToken` key a constant in this class.
You shouldn't return an empty string, it should be Optional so it's easier to check with `guard` and `if let` when using it.